templates/_helpers.tpl

```diff
@@ -77,12 +81,15 @@ application-credential-secret={{ include "auth_application_credential_secret" .
 username={{ include "auth_username" . }}
 password={{ include "auth_password" . }}
 user-domain-name={{ include "auth_user_domain_name" . }}
+domain-name={{ default (include "auth_user_domain_name" .) (include "auth_domain_name" .) }}
 tenant-id={{ include "auth_project_id" . }}
+project-id={{ include "auth_project_id" . }}
 {{ end }}
 
 [LoadBalancer]
+enabled={{ not (.Values.yawol | default false) }}
+use-octavia={{ not (.Values.yawol | default false) }}
```

Comment:
This setting has no effect, see https://github.com/SovereignCloudStack/k8s-cluster-api-provider/issues/722